### DISCUSSION TITLE
How to jump the sprint onTap?

### DISCUSSION THREAD

**Discussion by rddewan** [flame-engine/flame#2172] [Q&A] [answered]
Hi,
can anyone share the code to jump when user tap or double tap the screen ?

**Comment by ufrshubham** [comment, marked as answer]
Here are few places where you can find code for jump:
- https://docs.flame-engine.org/main/tutorials/platformer/platformer.html (Official tutorial from Flame)
- https://examples.flame-engine.org/#/Experimental_Follow%20and%20World%20bounds (Official camera example,implements jump)
- https://github.com/ufrshubham/dino_run
- https://github.com/ufrshubham/flame_simple_platformer
